src/hooks/useElementsOverflowingIndex.ts

```diff
@@ -1,14 +1,26 @@
-import { useCallback, useEffect, useState } from "react";
-import last from "lodash/last";
+import { ReactNode, RefObject, useCallback, useEffect, useState } from "react";
+import { last } from "lodash";
```

Comment:
Do we have proper tree-shaking to support this?

Reply:
as far as i know, yes, i can change it to esm

Reply:
@sahariko